app/constants/live.js

```diff
@@ -1,8 +1,8 @@
 export const ROLES = {
-  host: 'host',
-  maven: 'maven',
-  moderator: 'moderator',
-  guest: 'guest',
+  Host: 'host',
+  Maven: 'maven',
+  Moderator: 'moderator',
+  Guest: 'guest',
```

Comment:
Did you checked the scope of this constants, like what impact we can have if we change this constants?

Reply:
changed it. @satyam73 re-check now.